modules/bullet/rigid_body_bullet.cpp

```diff
@@ -834,7 +832,7 @@ void RigidBodyBullet::reload_shapes() {
 	btBody->updateInertiaTensor();
 
 	reload_kinematic_shapes();
-
+	set_continuous_collision_detection(btBody->getCcdMotionThreshold() < 10000.0);
```

Comment:
Isn't it a bit risky if that `10000.0` magic threshold ends up being `9999.999123` due to floating point precision errors?

Reply:
Fair